source/dagger/gameplay/team_game/team_game_collisions.h

```diff
@@ -35,6 +35,8 @@ namespace team_game
         Bool canGoUp{ true };
         Bool canGoDown{ true };
 
+        Bool grounded = false;
```

Comment:
don't see if u use it anywhere

Reply:
Oh, right, that's a remnant of an older idea, will be removed